src/versionbits.h

```diff
@@ -16,39 +16,47 @@ static const int32_t VERSIONBITS_TOP_BITS = 0x20000000UL;
 static const int32_t VERSIONBITS_TOP_MASK = 0xE0000000UL;
 /** Total bits available for versionbits */
 static const int32_t VERSIONBITS_NUM_BITS = 29;
+/** Size of window to use for assessing warning of unknown bits */
+static const int BIT_WARNING_WINDOW = 100;
+/** Threshold to use for assessing warning of unknown bits */
+static const int BIT_WARNING_THRESHOLD = 50;
 
+// bip135: assigned numbers to these enum values
 enum ThresholdState {
-    THRESHOLD_DEFINED,
```

Comment:
Agreed - will remove.
I might have wanted to set these to known values for debugging purposes.